src/modelset/downloader.py

```diff
@@ -7,16 +7,27 @@
 
 URL_ZIP_MODELSET = 'https://github.com/modelset/modelset-dataset/releases/download/v0.9.3/modelset.zip'
 DEFAULT_DIR_MODELSET = os.path.join(os.path.expanduser('~'), '.modelset', 'modelset')
+URL_WORD2VEC = 'http://sanchezcuadrado.es/files/modelset/vectors.kv'
+WORD2VEC_NAME_KV = 'vectors.kv'
 
 
-def main(args):
+def download_modelset(args):
     tempdir = tempfile.gettempdir()
     filename = wget.download(URL_ZIP_MODELSET, out=tempdir)
     shutil.unpack_archive(filename, extract_dir=tempdir)
     shutil.move(os.path.join(tempdir, "modelset"), args.output_dir)
     os.remove(filename)
 
 
+def download_word2vec(args):
+    wget.download(URL_WORD2VEC, out=args.output_dir)
+
+
+def main(args):
+    download_modelset(args)
```

Comment:
I suggest capturing an exception here, for each "download method" so that it one fails we output a message but at least we have downloaded the other. This is particularly important for download_word2vec since it is an different place than modelset-dataset (e.g., github vs sanchezcuadrado.es)

Reply:
Done.